src/map/pc.cpp

```diff
@@ -9235,6 +9235,18 @@ int pc_setregistry(struct map_session_data *sd, int64 reg, int val)
 	return 1;
 }
 
+//maybe move this as intif or script ?
```

Comment:
Should be usable/used by all modules that have script variables.

Reply:
yeah I think I'd just put it in intif as it's the one "blocking/sending" the data.
Plus I don't wanna bloated pc.c more then it's already is.